apps/web/components/observability/messages/message-content.test.tsx

```diff
@@ -14,6 +14,7 @@ jest.mock("@/components/ui/tambo/markdown-components", () => ({
 jest.mock("../utils", () => ({
   formatTime: (_date: Date) => "12:00 PM",
   getMessageContexts: () => [],
+  extractSelectedInteractablesFromAdditionalContext: () => [],
```

Comment:
it seems really strange to mock this out, as this is a pure function in our own code